patches/chrome-installer-util-google_update_settings.cc.patch

```diff
@@ -14,23 +14,48 @@ index 441ed6d50d5cb4edcd665665fca7887f034ff6a0..709c7e5e24518902ae86ab1ef819dd9d
  const wchar_t GoogleUpdateSettings::kUpdatePolicyValue[] = L"UpdateDefault";
  const wchar_t GoogleUpdateSettings::kDownloadPreferencePolicyValue[] =
      L"DownloadPreference";
-@@ -41,6 +45,11 @@ const wchar_t GoogleUpdateSettings::kCheckPeriodOverrideMinutes[] =
- const int GoogleUpdateSettings::kCheckPeriodOverrideMinutesMax =
-     60 * 24 * 7 * 6;
+@@ -43,7 +47,7 @@ const int GoogleUpdateSettings::kCheckPeriodOverrideMinutesMax =
  
-+
-+#if defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
-+#define GOOGLE_CHROME_BUILD
-+#endif
-+
  const GoogleUpdateSettings::UpdatePolicy
      GoogleUpdateSettings::kDefaultUpdatePolicy =
- #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
-@@ -836,3 +845,7 @@ bool GoogleUpdateSettings::ReadExperimentLabels(
- 
-   return result == ERROR_SUCCESS;
+-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
++#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || (defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD))  // NOLINT
+         GoogleUpdateSettings::AUTOMATIC_UPDATES;
+ #else
+         GoogleUpdateSettings::UPDATES_DISABLED;
+@@ -149,7 +153,7 @@ bool InitChannelInfo(bool system_install,
+   return channel_info->Initialize(key);
  }
-+
-+#if defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
-+#undef GOOGLE_CHROME_BUILD
-+#endif
+ 
+-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
++#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || (defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD))  // NOLINT
```

Comment:
this seems to actually do the opposite of what it did before and we're adding a bunch of patches for what was previous done in a few lines

Reply:
Fixed.